### PR TITLE
Feature/overhaul

### PR DESCRIPTION
**Pull request by myhndl**
includes Windows dev instructions and much, much more

### CODE REVIEW

**Comment by myhndl** [comment]
There's some TODOs throughout. Things I'll need help with:
1. linking TODOs - I just noted where they should be linked to
1. callouts of varying severity. I added some quote (`>`) at various places prefaced with `Info:` or `Warning:`. These need to be transformed into callouts of matching their severity level.

Not done yet:
1. All tutorials after EICAR. Given how pressing Windows instructions are, let's temporarily drop these additional tutorials.

**Comment by mrtizmoatwork** [comment]
@myhndl your dev_env_lin.md needs to instruct the user to install python3 and it MUST be a version newer than 3.5.2 (3.5.2 cannot run polyswarm-client due to bugs). Windows instructions specify 3.5.4.

To prevent us from having to deal with the myriad python3 versions on each linux distro, I think we should make a blanket statement that they need python3 > 3.5.2 and let them figure out their own instructions for how to install said python version. On my own linux dev env (ubuntu 16.04, I use python 3.6.7 and it works great thus far).

**Comment by mrtizmoatwork** [comment]
@myhndl in the tutorial sections, we need to totally remove the "testnet" and testing content since it differs between Windows/Linux and causes a distraction when the user should be focusing on writing the engine.

What we should do is have a single page where we have "here is how to test a linux engine" and another  page where we have "here is how to test a windows engine". Then, in the tutorials, we can just link to those pages and not have all of that content duplicated throughout, causing distractions and confusion.

**Comment by myhndl** [comment]
Addressed feedback:
1. Set minimum Python to 3.5.4, parity with Windows. Python 3.5.2 actually runs into some edge cases with web3 sometimes due to a bug that was fixed in subsequent versions.
1. Eliminated `dev_env_common.md`, moved instructions back to `_lin.md` and `_win.md`.
1. Refactored testing instructions out of tutorials, into a new file (`testing.md`).

File: public-src/_pages/privacy.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: t-terms
+title: pages.privacy.title
+description: pages.privacy.description
+permalink: /privacy/
+header: purple
+settings: true
+---
+
+{% tf _pages/privacy.md %}
```
[the remark this answers is not in the document]
Comment (mrjoshc) [reply public-src/_pages/privacy.md]: Unfortunately not because cookies don't work across domains.

[the remark this answers is not in the document]
Comment (mrjoshc) [reply public-src/_pages/privacy.md]: Or is there a way since it's a subdomain?

[the remark this answers is not in the document]
Comment (mrjoshc) [reply public-src/_pages/privacy.md]: @myhndl As discussed, I will do a separate PR that removes this page and iframes in the privacy policy so we don't need to duplicate this page.

**Pull request merged**